README.md

```diff
@@ -205,11 +205,34 @@ hatch test integration_tests
 
 ### Run `kagglehub` from source
 
+#### Option 1: Execute a one-liner of code from the command line
+
 ```sh
 # Download a model & print the path
 hatch run python -c "import kagglehub; print('path: ', kagglehub.model_download('google/bert/tensorFlow2/answer-equivalence-bem'))"
 ```
 
+#### Option 2: Run a saved script from the /tools/scripts directory
```

Comment:
Can you open a separate PR to add this feature: [go/small-cls](https://goto.google.com/small-cls)